node/src/neighborhood/node_location.rs

```diff
@@ -98,37 +91,25 @@ mod tests {
     }
 
     #[test]
-    fn create_gossip_node_with_addr_and_country_code_reveal_results_in_node_with_addr_adn_free_world_bit(
+    fn node_record_from_gossip_with_addr_and_country_is_populated_with_right_addr_and_free_world_bit(
     ) {
-        let mut node = make_node_record(2222, true);
-        let country_record = pick_country_code_record(2222);
-        println!("node: {:?}", node);
-        node.metadata.node_location_opt = Some(NodeLocation {
-            country_code: country_record.1.clone(),
-            free_world_bit: country_record.2,
-        });
-        node.metadata.node_addr_opt = Some(NodeAddr::new(&country_record.0, &[8000 % 10000]));
-        node.inner.country_code = country_record.1;
-        let db = db_from_node(&node);
+        let mut original_node_record = make_node_record(2222, true);
+
+        let db = db_from_node(&original_node_record);
         let builder = GossipBuilder::new(&db);
 
-        let builder = builder.node(node.public_key(), true);
+        let builder = builder.node(original_node_record.public_key(), true);
 
         let mut gossip = builder.build();
         let gossip_result = gossip.node_records.remove(0);
-        let node_record = NodeRecord::try_from(&gossip_result).unwrap();
+        let result_node_record = NodeRecord::try_from(&gossip_result).unwrap();
 
-        println!("node_record: {:?}", node_record);
-        assert_eq!(
-            gossip_result.node_addr_opt.unwrap(),
-            node.node_addr_opt().unwrap()
-        );
-        assert_eq!(node_record.inner.country_code, "US")
+        original_node_record.metadata.last_update = result_node_record.last_updated();
+        assert_eq!(result_node_record, original_node_record)
     }
 }
 
 #[allow(dead_code, unused_imports)]
```

Comment:
If we do want to preserve this module with the benchmarks, then you should probably replace this attribute above simply with the standard #[cfg(test)].

However, I realized that if we do that it will never SEE the fully populated country code database with all the 60K-lines data. 

Honestly, this is where we come to the need for a critical decision what would be right to do here. I do require to discuss this together with Dan.

The concern is that the benchmark makes an impression of a legit measurement but is fake. It seems like we ARE interested in the times of look-ups in the database. However, we test it on DIFFERENT data set, almost anecdotally   extremely restricted, that of course gives back lovely tiny measures of almost no time. But by these wanna-be tests we are laying ourselves uncovered.

If we decide collectively that we want to keep some kind of benchmarks in we should design it more carefully as it may not apply to a unit test anymore given the high expected time lapses. It may be quite a challenge. We'll see. **Dan's oversight required though**.